app/views/plans/_share_form.html.erb

```diff
@@ -103,7 +103,7 @@
       <h2><%= _('Request expert feedback') %></h2>
       <p><%= _('Click below to give data management staff at your organisation access to read and comment on your plan.') %></p>
       <div class="well well-sm">
-        <%= current_user.org.feedback_email_msg.html_safe %>
```

Comment:
Why remove the `.html_safe` here? Should we use `sanitize` or `raw` instead? This info comes off of the org edit page and is entered by users.

Reply:
Doh! I meant to add `to_s.html_safe` to protect against nil values. Well caught.